etip/trackers/tests.py

```diff
@@ -321,3 +324,155 @@ def test_with_trackers(self):
             ]
         }
         self.assertJSONEqual(f.getvalue(), expected_json)
+
+
+EXODUS_API_BASE_URL = 'https://reports.exodus-privacy.eu.org'
+EXODUS_API_BASE_PATH = '/api/trackers'
+
+
+class CompareTrackersWithExodusCommandTest(TestCase):
+
+    def test_api_gets_called_correctly(self):
+        with patch('requests.get') as mocked_get:
+            mocked_get.return_value.status_code = 200
+            out = StringIO()
```

Comment:
I'm wondering if we need to keep this line if we don't make any assertion of the output of the command.

Reply:
Defining an `stdout=` param in the `call_command` is useful to avoid having logs spit out in the console when executing tests (it overrides standard stdout), but it's true I don't need a named variable if I don't use it after. Corrected!